guide/13-managing-arcgis-applications/enterprise-sites-guide.ipynb

```diff
@@ -0,0 +1,586 @@
+{
```

Comment:
<p>I think it would help to add a link to the <a href="https://doc.arcgis.com/en/hub/content/content-basics.htm" target="_blank">Content Basics</a> doc for users to find more information on this.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1450/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>